bin/report_runs.py

```diff
@@ -133,6 +140,15 @@ def report_runs(run_ids, noemail=False):
                         round(clean_yield/1000000000, 1), int(sdata['required_yield']/1000000000),
                         round(mean_cov, 1), sdata['required_coverage']
                     )
+                # Checking for other run elements which are still pending
+                for sample_run_element in query_dict(sdata, 'run_elements') or []:
+                    # Splitting the run element, and generating the run_id by concatenating the first four components
+                    # with an underscore
+                    sample_run_element_sections = sample_run_element.split('_')
```

Comment:
This is not used

Reply:
Well spotted, this has now been removed.